src/js/content_scripts/input-mapper-base.js

```diff
@@ -28,100 +28,116 @@ https://github.com/fluid-lab/gamepad-navigator/blob/master/LICENSE
-                        speedFactor: 1
+                        speedFactor: 1,
+                        background: true
                     },
                     // Face Button.
                     // Circle on PlayStation controller & B on Xbox controller.
                     "1": {
                         defaultAction: null,
                         currentAction: null,
-                        speedFactor: 1
+                        speedFactor: 1,
+                        background: true
                     },
                     // Face Button.
                     // Square on PlayStation controller & X on Xbox controller.
                     "2": {
                         defaultAction: "previousPageInHistory",
                         currentAction: null,
-                        speedFactor: 1
+                        speedFactor: 1,
+                        background: true
                     },
                     // Face Button.
                     // Triangle on PlayStation controller & Y on Xbox controller.
                     "3": {
                         defaultAction: "nextPageInHistory",
                         currentAction: null,
-                        speedFactor: 1
+                        speedFactor: 1,
+                        background: true
                     },
                     // Left Bumper.
                     "4": {
                         defaultAction: "reverseTab",
                         currentAction: null,
-                        speedFactor: 2.5
+                        speedFactor: 2.5,
+                        background: true
                     },
                     // Right Bumper.
                     "5": {
                         defaultAction: "forwardTab",
                         currentAction: null,
-                        speedFactor: 2.5
+                        speedFactor: 2.5,
+                        background: true
                     },
                     // Left Trigger.
                     "6": {
                         defaultAction: null,
                         currentAction: null,
-                        speedFactor: 1
+                        speedFactor: 1,
+                        background: true
                     },
                     // Right Trigger.
                     "7": {
                         defaultAction: null,
                         currentAction: null,
-                        speedFactor: 1
+                        speedFactor: 1,
+                        background: true
                     },
                     // Select/Share on PlayStation controller & Back on Xbox controller.
                     "8": {
-                        defaultAction: null,
+                        defaultAction: "closeCurrentTab",
                         currentAction: null,
-                        speedFactor: 1
+                        speedFactor: 1,
+                        background: true
```

Comment:
This structure is getting complex enough that it's probably time to document what elements are supported and what each means.  I'll create a separate ticket with examples of how we've handled this in the past.

Reply:
Yes, I think a separate ticket would be sufficient for the purpose.